freqtrade/exchange/exchange.py

```diff
@@ -1153,6 +1153,37 @@ def get_balances(self) -> dict:
             balances.pop("total", None)
             balances.pop("used", None)
 
+            if self.trading_mode == TradingMode.FUTURES:
+
+                open_orders_response: List[dict] = self._api.fetch_open_orders()
+                open_orders: dict = {}
+                for order in open_orders_response:
+                    symbol: str = order['symbol']
+                    open_orders[symbol] = order
+
+                positions: List[dict] = self._api.fetch_positions()
+                for position in positions:
+                    symbol = position['symbol']
+                    market: dict = self.markets[symbol]
+                    size: float = self._contracts_to_amount(symbol, position['contracts'])
+                    side: str = position['side']
+                    if size > 0:
+
+                        if symbol in open_orders:
+                            order_amount: float = open_orders[symbol]['remaining']
+                        else:
+                            order_amount = 0
+
+                        if side == 'long' or side == 'buy':
```

Comment:
Is it fine if we don't do anything for shorts? Shorts mean your selling the base, which means you're getting the quote. This will be returned from the balance api endpoint as used quote (USDT), and we don't really want to say that it's free, because it's locked up in the position.